public/_redirects

```diff
@@ -148,13 +149,10 @@ https://cert-manager.io/v1.4-docs/aboutjetstack/* https://cert-manager.io/suppor
 https://cert-manager.io/v1.5-docs/aboutjetstack/* https://cert-manager.io/support/ 301!
 https://cert-manager.io/v1.6-docs/aboutjetstack/* https://cert-manager.io/support/ 301!
 https://cert-manager.io/v1.7-docs/aboutjetstack/* https://cert-manager.io/support/ 301!
-
-# After the website refresh, we added a new section to plug Jetstack, so the aboutjetstack page
-# looked out of place in the docs.
+https://cert-manager.io/v1.8-docs/aboutjetstack/* https://cert-manager.io/support/ 301!
 https://cert-manager.io/docs/aboutjetstack/* https://cert-manager.io/support/ 301!
+https://cert-manager.io/next-docs/aboutjetstack/* https://cert-manager.io/support/ 301!
```

Comment:
I tried https://deploy-preview-976--cert-manager.netlify.app/next-docs/aboutjetstack/ but it just gave me the 404 page rather than redirecting.
But so do the original redirects https://deploy-preview-976--cert-manager.netlify.app/v0.15-docs/aboutjetstack/ which I can see redirect properly when deployed to the production site: https://cert-manager.io/v1.5-docs/aboutjetstack/ 
...so perhaps these will just work.

Reply:
Yeah, I mentioned this in our standup; because we have "https://cert-manager.io" on the redirect, it forces the redirect to only take effect when the site is deployed to that domain and won't work on deploy previews.

https://github.com/cert-manager/website/pull/977 will fix this for deploy previews and also allow us to test redirects locally without needing a deploy preview!